legal-api/tests/unit/services/warnings/business/business_checks/test_corps.py

```diff
@@ -0,0 +1,58 @@
+# Copyright © 2024 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in business with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Test suite to ensure Corpse business checks work correctly."""
+import pytest
+from unittest.mock import patch, Mock
+from legal_api.models import Business, Filing
+from legal_api.services.warnings.business.business_checks import WarningType
+from legal_api.services.warnings.business.business_checks.corps import check_business
+
+@pytest.fixture
+def business():
+    """Create a mock business object for testing."""
+    return Business(identifier="BC1234567")
+
+@pytest.fixture
+def filing():
+    """Create a mock filing object for testing."""
+    mock_filing = Mock()
+    mock_filing.effective_date = None
+    mock_filing.payment_completion_date = None
+    return mock_filing
```

Comment:
no need to create a fixture for this. import from here `from tests.unit.models import factory_business, factory_completed_filing`